src/EditorFeatures/CSharpTest/IntroduceVariable/IntroduceLocalForExpressionTests.cs

```diff
@@ -391,6 +389,160 @@ void M()
     {
         int {|Rename:v|} = 1 + 1;
     }
+}");
+        }
+
+        [Fact, WorkItem(39537, "https://github.com/dotnet/roslyn/issues/39537")]
+        public async Task IntroduceDeconstruction1()
```

Comment:
❔ Is this _always_ deconstructed, or are both options presented for the user to pick?

Reply:
always deconstructed.

Reply:
Now the user can pick between one or the other.  Looks like this:

![image](https://user-images.githubusercontent.com/4564579/219498036-53fb6829-cebf-4640-955b-2e919d81e64b.png)